### PR TITLE
Add Linear

### PR DESCRIPTION
**Pull request by fmerian**
Linear is an issue and project tracking software built for developers.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,6 +115,7 @@ The definition of developer-first for this repo is:
 *Tools that help in tracking bugs.*
 * [Lightrun](https://www.lightrun.com) - Debug production servers using a plugin in the IDE (on-prem or SaaS) that supports Java, Node.js, and Python.
 * [Rookout](https://www.rookout.com) - Debug production servers via a cloud-based debugger. Supporting Java, .Net, Node.js, Python, and Ruby.
+* [Linear](https://linear.app/) - Issue tracking.
```
Comment (agamm) [inline README.md:118]:
```suggestion
* [Linear](https://linear.app/) - Issue tracking built for developers.
```

**Review by agamm — CHANGES_REQUESTED** [review]
Better?

File: README.md (hunk 1 above)
Comment (fmerian) [reply README.md]: Much better :)

**Review by agamm — APPROVED** [review]
🔥

**Pull request merged**